src/modules/handle_tasks.js

```diff
@@ -0,0 +1,105 @@
+export default class HandleTasks {
+  constructor(store, UI) {
+    this.store = store;
+    this.UI = UI;
+  }
+
+  addItem() {
+    this.input = document.getElementById('add-task');
+    this.arrowIcon = document.querySelector('.arrow-icon');
+
+    this.input.addEventListener('keydown', (e) => {
+      const i = this.store.getItems().length;
+
+      if (e.key === 'Enter' && e.target.value) {
+        const obj = {
+          description: e.target.value,
+          completed: false,
+          index: i,
+        };
+        this.input.value = '';
+        this.store.add(obj);
+        this.UI.generateHtmlTasks();
```

Comment:
You could create a function to store these lines, you reuse the exact same ones in lines 35-37.

Reply:
- [x] change implemented